src/lib/viewers/controls/annotations/annotations-button-region-tooltips/PersistentOnboardingBoxEditAnnotationsTooltip.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useContext, PropsWithChildren } from 'react';
+import classNames from 'classnames';
+
+// TODO: Compare bundle sizes and possibly re-write here
+import { TargetedClickThroughGuideTooltip } from 'box-ui-elements/es/features/targeting';
+
+import { currentLocale } from '../../../../i18n';
+
+import { TargetingApi } from '../../../../types';
+
+import { ControlsLayerContext } from '../../controls-layer';
+import { ExperiencesContext } from '../../experiences';
+
+import './PersistentOnboardingBoxEditAnnotationsTooltip.scss';
+
+const LOCALE_LANGUAGE_ENGLISH = 'en';
+
+export type Props = PropsWithChildren<{
+    isEnabled?: boolean;
+}>;
+
+export default function PersistentOnboardingBoxEditAnnotationsTooltip({
+    children,
+    isEnabled = false,
+}: Props): JSX.Element {
+    const { experiences } = useContext(ExperiencesContext);
+    const { setIsForced } = useContext(ControlsLayerContext);
+
+    const {
+        persistentOnboardingBoxEditAnnotations: experience = {
```

Comment:
Why do we need this default value?

Reply:
Because I marked each experience as optional on `Experiences` type. Likely it wasn't a good decision. I explained it more here: https://github.com/box/box-content-preview/pull/1445#discussion_r871424732